### DISCUSSION TITLE
MP4, GIF instead of .jpg, .png

### DISCUSSION THREAD

**Discussion by Plinko666** [berrysauce/basketball#18] [Ideas]
Hello, I think its will be great to add possibility to export from Dribbble also mp4, gif formats. I really enjoy your project and wanna use it if you can help me with that feature.

**Comment by berrysauce** [comment]
Sure I'll look into it. Also GIFs are already supported 😉 Created #19

**Reply by Plinko666** [reply]
Its not export gif shots :( they come as still img not GIF
